internal/model/board.go

```diff
@@ -27,61 +37,114 @@ type Board struct {
 
 	// RemainingTile is the tile that was not placed on the board.
 	RemainingTile *BoardTile `json:"remainingTile"`
+
+	// Players holds the players that are part of the game.
+	Players []*Player `json:"players"`
+
+	// GameState is the current game state
+	State GameState `json:"gameState"`
+}
+
+func (b Board) tryPlaceTile(index int) error {
```

Comment:
Why a lowercase "T" while other function names start with a capital letter?

Also, the word "try" is weird. I see this function as a validator, so why not call it `validateTileMove` instead?

Reply:
The lower-case letter is used to define private methods.

I initialy used `try` to make a parallel with exception handling, but `validate` is definitly clearer. Thank you !